ckanext/datavic_iar_theme/templates/package/snippets/package_form.html

```diff
@@ -1,22 +1,58 @@
-{% ckan_extends %}
```

Comment:
Copied the whole package_form.html from the core since error_message was placed below the form and not inside any block.